src/main/java/seedu/address/model/person/DisplayPhoto.java

```diff
@@ -0,0 +1,97 @@
+package seedu.address.model.person;
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+//@@author keithsoc
+/**
+ * Represents a Person's Display Photo in the address book.
+ * Guarantees: immutability and validity.
+ */
+public class DisplayPhoto {
+
+    public static final String MESSAGE_PHOTO_CONSTRAINTS = "Display photo: "
+            + "specified file does not exist or it exceeded maximum size of 1MB.";
+
+    /* Display photos for sample persons in SampleDataUtil */
+    public static final String SAMPLE_PHOTO_ALEX = "/images/sample_dp_one.png";
+
+    private static final String DEFAULT_SAVE_DIR = "data/";
+    private static final int MAX_SIZE = 1000000; // Sets allowable maximum display photo size to 1MB
+
+    public final String value;
+
+    /**
+     * Validates given Display Photo.
+     *
+     * @throws IllegalValueException if given display photo string is invalid.
+     */
+    public DisplayPhoto(String photoPath) throws IllegalValueException {
+        if (photoPath == null || photoPath.isEmpty()) { // Allow null values
+            this.value = null;
+        } else if (photoPath.equals(SAMPLE_PHOTO_ALEX)) { // Display photos for sample persons
```

Comment:
What is the sample photo for?

Reply:
For `SampleDataUtil.java`!

Reply:
should we set this in SampleDataUtil then?

Reply:
Oh! This check has to be in this class because the path for sample photos and path user specified are completely different. Javafx could load `/images/...` but it requires `file://...` for user specified images.